src/core/server/saved_objects/migrations/actions/constants.ts

```diff
@@ -11,7 +11,11 @@
  * Uses the default value of 1000 for Elasticsearch reindex operation.
  */
 export const BATCH_SIZE = 1_000;
-export const DEFAULT_TIMEOUT = '60s';
+/**
+ * The default Elasticsearch client requestTimeout is 30s so we choose a
+ * shorter timeout to avoid the client aborting our requests before we receive a response
+ */
+export const DEFAULT_TIMEOUT = '29s';
```

Comment:
From what I understand we are using `29s` because it is below `30s` request timeout. Maybe we'd increase the reliability of this by reducing it to `28s` or so since event loop delays will affect when the timeout is triggered and a 1-second delay is not that uncommon if the process is busy.

Reply:
yes, I had a local commit that I forgot to push 🙈 but this was exactly what I was worried about. So the PR now sets a requestTimeout of 120s and DEFAULT_TIMEOUT=60s so it's very unlikely that event loop delays, dns lookups or other network conditions causes Kibana to cancel the request before ES had 60s to respond.